File: python/usageExample.py
from orangeSMS import sendSMS, getToken


clientID = "aklGYUdsYjdhY2VpM...Hkxdg=="
token = getToken(clientID)
senderAdress = "+2210000"
address = "+221781234567"

result = sendSMS(senderAdress, address, "Salam from python", token)
print(result)

File: javascript/usageExample.js
import {getToken, sendSMS} from './orangeSMS.js';

const clientID = "aklGYUdsYjdhY...QVRBOVVkVTFqRHkxdg=="
const token = await getToken(clientID)
let senderAdress = "+2210000"
let address = "+221781234567"

sendSMS(senderAdress, address, "bonjour from node", token)
    .then(result => {
        console.log(result);
    });